docs/services/base-rc/_index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Base Remote Control Service"
+linkTitle: "Base Remote Control"
+weight: 80
+type: "docs"
+description: "The Base Remote Control Service implements remote control for a base."
+tags: ["base", "services", "rover", "input controller", "remote control"]
+# icon: "/services/img/icons/base-remote-control.svg"
+# images: ["/services/img/icons/base-remote-control.svg"]
+# SME: Eric
+---
+
+The Base Remote Control service implements an [input controller](/components/input-controller/) as a remote control for a [base](/components/base/).
+This uses the [`input` api](/components/input-controller/#api) to make it easy to add remote drive controls for your rover or other mobile robot with a controller like a gamepad.
+
+Add the Base Remote Control service after configuring your robot with a base and input controller to control the linear and angular velocity of the base with the controller's button or joystick controls.
+
+Control mode is determined by the configuration attribute `"mode"`, for which there are five options:
+
+1. `"arrowControl"`: Arrow buttons control speed and angle
+2. `"triggerSpeedControl"`: Trigger button controls speed and joystick controls angle
+3. `"buttonControl"`: Four buttons (usually X, Y, A, B) control speed and angle
+4. `"joyStickControl"`: One joystick controls speed and angle
+5. `"droneControl"`: Two joysticks control speed and angle
+
+Monitor the input from these controls in the **Control** tab of the [Viam app](https://app.viam.com).
+
+## Configuration
+
+You must configure a [base](/components/base/) with a [movement sensor](/components/movement-sensor/) as part of your robot to configure a Base Remote Control service.
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Services** subtab and navigate to the **Create service** menu.
+Select the type `Navigation` and enter a name for your service.
+
+Click **Create service**:
+
+![An example configuration for a Base Remote Control service in the Viam app Config Builder.](img/base-rc-ui-config.png)
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
```

Comment:
Please follow the style we use for other templates

Reply:
My bad, I think done!

Reply:
yep perfect!